Portfolio/index.html

```diff
@@ -35,37 +35,64 @@ <h1>SKILLS</h1>
          <span>
         <img src="../html.icon.png" alt="HTML image" ><div>HTML</div><br>
          <img src="../css.icon.png" alt="CSS image"><div>CSS</div><br>
-         <img src="../javaScript.icon.png" alt="JAVASCRIPT image"><div>JAVASCRIPT</div><br>
-         <img src="../typescript.icon.png" alt="TYPESCRIPT image"><div>TYPESCRIPT</div><br>
+         <img src="../js.icon.png" alt="JAVASCRIPT image"><div>JAVASCRIPT</div><br>
+         <img src="../ts.icon.png" alt="TYPESCRIPT image"><div>TYPESCRIPT</div><br>
          <img src="../svelte.icon.png" alt="SVELTE image"><div>SVELTE JS</div><br>
          <img src="../firebase.icon.png" alt="FIREBASE image"><div>FIREBASE</div><br>
          <img src="../react.icon.png" alt="REACT image"><div>REACT<br>REACT NATIVE</div>
          </span>
-         <h3>GET IN TOUCH</h3>

```

Comment:
The closing </form> tag is unnecessary and should be removed to avoid invalid HTML structure.